qa/workunits/cephtool/test.sh

```diff
@@ -1111,6 +1111,195 @@ function test_mon_mon()
   expect_false ceph mon feature set abcd --yes-i-really-mean-it
 }
 
+function gen_secrets_file()
+{
+  # lets assume we can have the following types
+  #  all - generates both cephx and lockbox, with mock dm-crypt key
+  #  cephx - only cephx
+  #  no_cephx - lockbox and dm-crypt, no cephx
+  #  no_lockbox - dm-crypt and cephx, no lockbox
+  #  empty - empty file
+  #  empty_json - correct json, empty map
+  #  bad_json - bad json :)
+  #
+  local t=$1
+  if [[ -z "$t" ]]; then
+    t="all"
+  fi
+
+#  fn=$(mktemp -p $TEMP_DIR)
+  fn=$(mktemp -p /tmp/muh-test)
```

Comment:
```
mktemp: failed to create file via template '/tmp/muh-test/tmp.XXXXXXXXXX': No such file or directory
```

it does not work under some circumstances: 

- `/tmp/muh-test` does not exist at all
- some other tests might want to remove `/tmp/muh-test` without notice. 

why not use `mktemp $TEMP_DIR/muh-test.XXX` ?